.tx/config.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2018-2023 Nextcloud GmbH and Nextcloud contributors
+SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
GPL or MIT, the later would match the libs license

Reply:
Ah, right. I c&p from SSO.